e2e/vmss.go

```diff
@@ -115,45 +116,56 @@ func extractLogsFromVMLinux(ctx context.Context, s *Scenario) {
 	require.NoError(s.T, err)
 
 	commandList := map[string]string{
-		"cluster-provision":            "cat /var/log/azure/cluster-provision.log",
-		"kubelet":                      "journalctl -u kubelet",
-		"cluster-provision-cse-output": "cat /var/log/azure/cluster-provision-cse-output.log",
-		"sysctl-out":                   "sysctl -a",
-		"aks-node-controller":          "cat /var/log/azure/aks-node-controller.log",
+		"cluster-provision.log":            "cat /var/log/azure/cluster-provision.log",
+		"kubelet.log":                      "journalctl -u kubelet",
+		"cluster-provision-cse-output.log": "cat /var/log/azure/cluster-provision-cse-output.log",
+		"sysctl-out.log":                   "sysctl -a",
+		"aks-node-controller.log":          "cat /var/log/azure/aks-node-controller.log",
 	}
 
-	podName, err := getHostNetworkDebugPodName(ctx, s.Runtime.Cluster.Kube, s.T)
+	pod, err := s.Runtime.Cluster.Kube.GetHostNetworkDebugPod(ctx, s.T)
 	if err != nil {
 		require.NoError(s.T, err)
 	}
 
 	var logFiles = map[string]string{}
 	for file, sourceCmd := range commandList {
-		s.T.Logf("executing command on remote VM at %s: %q", privateIP, sourceCmd)
-
-		execResult, err := execOnVM(ctx, s.Runtime.Cluster.Kube, privateIP, podName, string(s.Runtime.SSHKeyPrivate), sourceCmd)
+		execResult, err := execOnVM(ctx, s.Runtime.Cluster.Kube, privateIP, pod.Name, string(s.Runtime.SSHKeyPrivate), sourceCmd)
 		if err != nil {
-			s.T.Logf("error fetching logs for %s: %s", file, err)
+			s.T.Logf("error executing %s: %s", sourceCmd, err)
 			continue
 		}
 		if execResult.stdout != nil {
 			out := execResult.stdout.String()
 			if out != "" {
-				logFiles[file+".stdout.txt"] = out
+				logFiles[file] = out
 			}
 
 		}
 		if execResult.stderr != nil {
-			out := execResult.stderr.String()
+			out := removeWarningMessages(execResult.stderr.String())
 			if out != "" {
-				logFiles[file+".stderr.txt"] = out
+				s.T.Logf("%q: %s", sourceCmd, out)
 			}
 		}
 	}
 	err = dumpFileMapToDir(s.T, logFiles)
 	require.NoError(s.T, err)
 }
 
+// removeWarningMessages removes noise from logs
+func removeWarningMessages(input string) string {
```

Comment:
looks like more then WArnings are being removed, shoudl the function name be removed ?

Reply:
What do you mean?

Reply:
deleted the function